pkg/resolver/service/service.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package service
+
+import (
+	"fmt"
+	"strings"
+	"unicode"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethersphere/bee/pkg/logging"
+	"github.com/ethersphere/bee/pkg/resolver"
+	"github.com/ethersphere/bee/pkg/resolver/client/ens"
+)
+
+// Service is the name resolution service ready for integration with bee.
+type Service struct {
```

Comment:
It looks to me like that Service is redundant to Multiresolver. This package provides the service only to have a Connect method doing something. Could the Service type be removed and only have a Connect and Parse* functions in multiresolver as that all is in function for it. After that the whole service package is no longer needed.

Reply:
I agree, I will remove the `Service` struct. Probably the `MultiResolver` is sufficient on it's own.